README.md

```diff
@@ -1,8 +1,42 @@
 # rapid-watershed-delineation
 
-A fork of @nazmussazib's [Rapid Watershed Delineation](https://github.com/nazmussazib/RapidWatersheDelineation) project, for use in [Model My Watershed](https://github.com/WikiWatershed/model-my-watershed).
+A docker image and fork of @nazmussazib's [Rapid Watershed Delineation](https://github.com/nazmussazib/RapidWatersheDelineation) project, for use in [Model My Watershed](https://github.com/WikiWatershed/model-my-watershed).
 
-A Docker container of this repo and it's dependencies is available at [docker-rwd](https://github.com/WikiWatershed/docker-rwd).
+[![Travis CI](https://api.travis-ci.org/WikiWatershed/docker-rwd.svg "Build Status on Travis CI")](https://travis-ci.org/WikiWatershed/docker-rwd/)
+[![Docker Repository on Quay.io](https://quay.io/repository/wikiwatershed/rwd/status "Docker Repository on Quay.io")](https://quay.io/repository/wikiwatershed/rwd)
+[![Apache V2 License](http://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/wikiwatershed/rapid-watershed-delineation/blob/develop/LICENSE)
+
+### Getting started
+* Define environment variables (see below)
+* Run `./scripts/update.sh`
+* Run `./scripts/server.sh`
+
+#### Linux
+* Run `curl http://localhost:5000/rwd/-75.276639/39.892986`
+
+#### OS X
+* Find the IP of the default VM using `docker-machine ip default`
+* Run `curl http://<default_vm_ip>:5000/rwd/-75.276639/39.892986`
+
+### Environment Variables
+
+| Name       | Description                                  | Example                             |
+| ---------- | -------------------------------------------- | ----------------------------------- |
+| `RWD_DATA` | Path to RWD data                     | `/media/passport/rwd-nhd`                   |
```

Comment:
This might be a formatting error in the original repo. Consider lining these columns up.